monitoring/snippets/v3/uptime-check-client/snippets_test.py

```diff
@@ -59,13 +58,13 @@ def uptime():
         yield uptime
 
 
-def test_create_and_delete(capsys):
+def test_create_and_delete(capsys) -> None:
     # create and delete happen in uptime fixture.
     with UptimeFixture():
         pass
```

Comment:
What is being tested here? Shouldn't a test check that the fixture yielded a proper value?

Reply:
I believe it tests Fixture for not throwing exceptions.